config/apiserver/rbac/user_clusterrole.yaml

```diff
@@ -0,0 +1,13 @@
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: bulward:user
```

Comment:
I this is name will be prefixed with `bulward-apiserver-` so it doesn't make sense to have something like:
`bulward-apiserver-bulward:user`.

Reply:
It won't be prefixed, Cluster objects aren't prefixed (I've checked via `kubectl get clusterroles`)